### DISCUSSION TITLE
Ender-3 S1 Setup

### DISCUSSION THREAD

**Discussion by jiriteach** [mriscoc/Ender3V2S1#92] [Q&A] [answered]
Hi - Thanks for the work on this firmware.
I have installed it on my Ender3-S1.

I just need some guidance - 
1. I dont need to change the offset like I would have done with the BLtouch? This is already done right?
2. I have done an Auto Build Mesh and the mesh is unlevel. I wouldnt have seen this with the stock firmware. Should I manually adjust?
3. Do I need to add the gcode to load the mesh?

Thanks

**Comment by mriscoc** [comment, marked as answer]
Hi, 
1. The X and Y probe offsets are the official offsets for a Stock S1. The Z probe offset needs to be adjusted using the z probe wizard (Prepare menu).
2.  Before of the Auto Build Mesh, do a bed tramming, then the auto build mesh and save it.
3.  Adds the M420 S1 as suggest here: https://github.com/mriscoc/Ender3V2S1/wiki/3D-BLTouch

**Reply by vkhoi80** [reply]
can you help me how to auto bed level, i don't see it. thanks

**Reply by mriscoc** [reply]
Menu Prepare, Build Mesh or Auto level

**Comment by jiriteach** [comment]
Awesome - thank you!
